src/Ztm.WebApi/ServicesExtensions.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.Extensions.DependencyInjection;
+using Ztm.Zcoin.NBitcoin.Exodus;
+
+namespace Ztm.WebApi
+{
+    public static class ServicesExtensions
```

Comment:
The name of extension class should start with what it extend. So in this case should be `ServiceCollectionExtensions`.